app/models/manageiq/providers/autosde/storage_manager/host_initiator.rb

```diff
@@ -11,14 +11,19 @@ def self.raw_create_host_initiator(ext_management_system, options = {})
     iqns = Array(options['iqn'])
     iqn_values = iqns.join(":")
 
+    def self.parse_cluster_name(cluster_name)
+      cluster_name == '<None>' ? '' : cluster_name
+    end
+
```

Comment:
Need to move this out of the `raw_create_host_initiator` method definition.  Also it probably should be a `private_class_method`

Reply:
the code climate check suggested using a lambda, hope that's ok

Reply:
Honestly I think a simple local variable would be easier